src/components/buttonsIndicators/button/index.tsx

```diff
@@ -21,6 +21,7 @@ export interface Props {
   onClick?: () => void
   id?: string
   disabled?: boolean
+  center?: boolean
```

Comment:
Please do not add custom props to standard components. Center it in the parent component

Reply:
Will do